### ISSUE TITLE
Problem with Video Slider

### ISSUE THREAD

**Issue by SalahAdDin** [smashingboxes/OwlCarousel2#997]
Hi guys, i was trying do a mixed video and image slider, i did step by step, and finally, the slider didn't work.
I have this: 
![nohayvideo](https://cloud.githubusercontent.com/assets/5034215/9395637/3b2a5274-4755-11e5-9da3-e20875637baa.png)
Can anyone help me?
I don't understand the merge option :'(

**Comment by JoergPf** [comment]
Same problem for me, I embed a video with 
     <div class="item-video" >
                          <a class="owl-video" href="http://vimeo.com/23924346"></a>
     </div>
 
But all I see is the playicon sitting somewhere in the top, like @SalahAdDin shows. 

Where could be the problem?

**Comment by JoergPf** [comment]
It seems to be a CSS problem - when I write

```html
.owl-carousel .owl-video-tn {
    min-height: 685px;
}
```

the video shows up. But I would like to be this responsive and auto...

**Comment by SalahAdDin** [comment]
But, it's suppouse that we should see the video thumbnail back to the play video image.

Can you see it?

**Comment by JoergPf** [comment]
Yes, the video posterframe is there, so I can see the image with the play icon on top.

**Comment by SalahAdDin** [comment]
Please, can you do a pull request?

**Comment by JoergPf** [comment]
I'm not so experienced with github - what is the purpose of doing a pull request?

**Comment by SalahAdDin** [comment]
yor fix, but, i'll test your fix and do a PR in your name :D, if you want.

**Comment by WPThemer** [comment]
The problem is with mix gallery height:100%; does not work until you set a fix height.. has someone found a fix yet ?..

**Comment by SalahAdDin** [comment]
I hadn't work with it, have you the same problem? Try do it for fix?

**Comment by adampatterson** [comment]
Related to #936

**Comment by quienti** [comment]
Still exiting in 2.1.0

**Comment by dham-tharan** [comment]
We can over come with CSS.

`.item-video {
    height: 450px; 
    width: auto;
    margin: 0 auto;
}`
